src/bundle/Resources/public/js/scripts/admin.role_assignment.add.js

```diff
@@ -78,11 +78,14 @@
     };
     const toggleDisabledState = () => {
         limitationsRadio.forEach((radio) => {
-            const disableNode = doc.querySelector(radio.dataset.disableSelector);
-            const methodName = radio.checked ? 'removeAttribute' : 'setAttribute';
+            const disableNode = radio.closest('.ibexa-assign__limitations-item').querySelector(radio.dataset.disableSelector);
```

Comment:
Actually, why don't we just use `disabled` attribute instead of custom `ibexa-dropdown--disabled` class?
(I know that this question is outside of the scope of this task.)